### ISSUE TITLE
Better UI to create RSS links (+ OPML export)?

### ISSUE THREAD

**Issue by georgjaehnig** [DIYgod/RSSHub#6024]
### What feature is it?

First, this is a great project! Thank you for all the work that was put in already.

I wonder if there already exists another project with a better UI that would make it easier to create RSS URLs?

For instance, instead of reading docs https://docs.rsshub.app/, I could have

- a list of available services, like the icons Twitter and Telegram etc.
- after a click an input field to enter a username
- have below automatically the RSS feed link generated

### What problem does this feature solve?

Currently, users have to create all RSS links by hand: Copy the template URLs and edit them with their data. This works, but it is not very user friendly.

Besides, I have to create, copy and import every new RSS URL separately. This can take a lot of time if I e.g. want to add a lot of RSS feeds at once.

Instead, it would be much faster to e.g.
- add a list of Twitter users or keywords I want to "follow"
- get one [OPML](https://en.wikipedia.org/wiki/OPML) file exported including all the new RSS feeds, which I can import into my feed reader in 1 click.  

Does such a project already exist?

**Comment by XiaoHuGuDong** [comment]
For better way to create RSS links, you could use RSSHub Radar (Browser Extensions) and RSSBud on ios14. You can find it in home page from docs.rsshub.app. But the only defect may be English localization missing. If so, you can open an issue.

**Comment by georgjaehnig** [comment]
Yes, thank you, I know about both projects. But if understand them correctly, for each new feed I have to visit the original website first, right?

So for instance, if I want to have Twitter keyword searches for

- apple
- banana
- pear

I have create every search of it first, and then get every RSS feed URL one by one.

Instead, I would like to drop that list somewhere, and in one go get the list of RSS URLs (e.g. as OPML).

If no such project does exist yet, I might start it. Just asking here to not reinvent the wheel. :)

**Comment by HenryQW** [comment]
I use iOS shortcuts to do it.

**Comment by NeverBehave** [comment]
We don’t have plans on related stuff but we are looking forward we could have something like that. Definitely start one if you’d like to.

**Comment by dvikan** [comment]
I have a related question regarding this. I would like to have a feature where I can paste:

https://gab.com/realdonaldtrump/

and automatically get the following feed:

https://rsshub.app/gab/user/realdonaldtrump

Does such a feature exist? Is it possible to automate this using the existing regexes?

My motivation for this is I've created an rss feed discovery service:

https://discovery.thirdplace.no/?q=https://gab.com/realdonaldtrump/

And I would like to make all rsshub routes discoverable.

**Comment by NeverBehave** [comment]
Used to have one, you may tried to make use Radar rules for this functions. We have enforced script to provide related discovery rules when submitted. 



Related: https://github.com/NeverBehave/RSShub-Generator
https://github.com/DIYgod/RSSHub-Radar/issues/692
